src/main/java/io/appium/java_client/remote/options/BaseMapOptionData.java

```diff
@@ -33,6 +35,11 @@ public BaseMapOptionData(Map<String, Object> options) {
         this.options = options;
     }
 
+    public BaseMapOptionData(String json) {
+        //noinspection unchecked
+        this((Map<String, Object>) new Gson().fromJson(json, Map.class));
```

Comment:
I guess it would be better to convert `Gson` object to a static field and reuse it here and in methods `toJson()` and `toString()`

Reply:
done